src/workerHelpers.no-bundler.js

```diff
@@ -55,7 +55,12 @@ export async function startWorkers(module, memory, builder) {
       const worker = new Worker(import.meta.url, {
         type: 'module'
       });
-      worker.postMessage(workerInit);
+      try {
+        worker.postMessage(workerInit);
+      } catch(e) {
+        console.warn("Have you added the `.cargo/config` and `rust-toolchain` files as explained in the wasm-bindgen-rayon readme?");
```

Comment:
It might be useful to reword this more generally, because, while adding `.cargo/config` is the easiest way, we still want to encourage devs to use feature detection, in which case they should follow different steps and pass config via env variables.

Maybe:

```suggestion
        console.warn('Have you following the "Setting up" and "Building Rust code" steps as described in the the wasm-bindgen-rayon readme?');
```